src/common-ui/components/Tags.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import localStyles from './TagOption.css'
+
+const Tags = ({
+    children,
+    onTagSearchChange,
+    setInputRef,
+    onTagSearchEnter,
+    numberOfTags,
+    handleClick,
+}) => (
+    <div className={localStyles.tagDiv}>
+        <form className={localStyles.searchContainer}>
+            <input
+                className={localStyles.search}
+                name="query"
+                placeholder="Search & Add Tag(s)"
+                onChange={onTagSearchChange}
```

Comment:
Any `<input>` where you're overriding `value` or `onChange` props are "controlled". As in they don't try to manage their state anymore, so you should do it somewhere. You should add a string state (redux or popup container) and pass in here as `value` prop and make sure `onTagSearchChange` is updating that state with the new values somehow (similar to how you're updating `newTag` now).